src/builtins/bg.rs

```diff
@@ -46,16 +41,20 @@ fn send_to_bg(
 }
 
 /// Builtin for putting a job in the background.
-pub fn bg(parser: &Parser, streams: &mut IoStreams, args: &mut [&wstr]) -> Option<c_int> {
+pub fn bg(parser: &Parser, streams: &mut IoStreams, args: &mut [&wstr]) -> c_int {
     let opts = match HelpOnlyCmdOpts::parse(args, parser, streams) {
         Ok(opts) => opts,
-        Err(err @ Some(_)) if err != STATUS_CMD_OK => return err,
+        Err(err) if err != STATUS_CMD_OK => return err,
         Err(err) => panic!("Illogical exit code from parse_options(): {err:?}"),
     };
 
-    let cmd = args[0];
+    let cmd = match args.get(0) {
+        Some(cmd) => cmd,
+        None => return STATUS_INVALID_ARGS,
+    };
```

Comment:
Very fly-by comment: this can be written more succinctly as:

```suggestion
    let Some(cmd) = match args.get(0) else {
        return STATUS_INVALID_ARGS;
    };
```